var/spack/repos/builtin/packages/pism/package.py

```diff
@@ -121,23 +122,7 @@ def setup_run_environment(self, env):
 # > Do you have handy a table of which versions of PETSc are required
 # > for which versions of PISM?
 #
-# We don't. The installation manual [1] specifies the minimum PETSc
-# version for the latest "stable" release (currently PETSc 3.3). The
-# stable PISM version should support all PETSc versions starting from the
-# one specified in the manual and up to the latest PETSc release.
-#
-# The current development PISM version should be built with the latest
-# PETSc release at the time (the "maint" branch of PETSc).
-#
-# Thanks to Git it is relatively easy to find this info, though:
-#
-# | PISM version | PETSc version |
-# |--------------+---------------|
-# |          0.7 | 3.3 and later |
-# |          0.6 | 3.3           |
-# |       new_bc | 3.4.4         |
-# |          0.5 | 3.2           |
-# |          0.4 | 3.1           |
-# |          0.3 | 2.3.3 to 3.1  |
-# |          0.2 | 2.3.3 to 3.0  |
-# |          0.1 | 2.3.3-p2      |
+# We don't. The current PISM version should be built with the latest
+# PETSc release <= 3.20 because of deprecation of `DMDASNESFunction`
```

Comment:
PISM v2.1 requires PETSc <= 3.20, but v2.1.1 I just tagged supports PETSc 3.21 and 3.22 as well.